base/spec/spec_helper.rb

```diff
@@ -2,6 +2,12 @@
 
 require "rspec/its"
 
+begin
+  require "pry-byebug"
+rescue LoadError
+  # we install pry-byebug only on MRI and truffleruby platforms
+end
+
```

Comment:
I'm not sure that's necessary, but if it doesn't work without it, I'm fine with it.